python/tbb/pool.py

```diff
@@ -89,8 +89,8 @@ class Pool(object):
 
     def __init__(self, nworkers=0, name="Pool"):
         """
-        \param nworkers (integer) number of worker threads to start
-        \param name (string) prefix for the worker threads' name
+        \\param nworkers (integer) number of worker threads to start
```

Comment:
Can you consider using sphinx-style syntax like in https://github.com/oneapi-src/oneTBB/blob/master/python/tbb/__init__.py#L172? 
\param -> :param

Reply:
I am more inclined to `@param` as described above, but I can change it to a colon.